### PR TITLE
HELM-402: Update system requirements in documentation

### PR DESCRIPTION
**Pull request by Bonrob2**
Updated system requirements, and changes requested in  https://github.com/OpenNMS/grafana-plugin/pull/669, which was merged too soon.

# Pull Request Check Sheet

* [ ] Have you read and followed our [Contribution Guidelines](https://github.com/OpenNMS/opennms/blob/develop/CONTRIBUTING.md)?
* [ ] Have you [made an issue in the OpenNMS issue tracker](https://issues.opennms.org)?<br>If so, you should:
  1. update the title of this PR to be of the format: `${JIRA-ISSUE-NUMBER}: subject of pull request`
  2. update the JIRA link at the bottom of this comment to refer to the real issue number
  3. prefix your commit messages with the issue number, if possible
* [ ] Have you made a comment in that issue which points back to this PR?
* [ ] Have you updated the JIRA link at the bottom of this comment to link to your issue?
* [ ] If this is a new feature, is there documentation?
* [ ] If this is a new feature or substantial change, are there tests that cover it?

# Pull Request Process

One or more reviewers should be assigned to each PR.

If you know that a particular person is subject matter expert in the area your PR affects, feel free to assign one or more reviewers when you create this PR, otherwise reviewers will be assigned for you.

Once the reviewer(s) accept the PR and the branch passes continuous integration in Bamboo, the PR is eligible for merge.

At that time, if you have commit access (are an OpenNMS Group employee or a member of the Order of the Green Polo) you are welcome to merge the PR.
Otherwise, a reviewer can merge it for you.

Thanks for taking time to contribute!

# External References

* JIRA (Issue Tracker): http://issues.opennms.org/browse/HELM-402
* Continuous Integration: [CircleCI](https://circleci.com/gh/OpenNMS/grafana-plugin)

### CODE REVIEW

File: docs/modules/installation/pages/requirements.adoc [hunk 1]
```diff
@@ -5,11 +5,12 @@ This section describes the requirements to install the {product-name}.
 
 == Runtime requirements
 
-For runtime operations, you need to install and set up either https://www.opennms.org[OpenNMS Horizon 29+] or https://www.opennms.com[Meridian 2021+].
+For runtime operations, you need to install and set up either https://www.opennms.com/horizon/[OpenNMS Horizon 31+] or https://www.opennms.com/[Meridian 2023+].
 You also need a http://docs.grafana.org/installation[Grafana instance] (version {grafana-version-required} or newer).
 
-NOTE: The plugin has been tested with Grafana versions up to {grafana-version-tested}.
-We recommend that you use matching versions of the plugin and Grafana (for example, {product-name} 8.x and Grafana 8.x).
+NOTE: We have tested he plugin with Grafana version {grafana-version-tested}.
```
Comment (Bonrob2) [inline docs/modules/installation/pages/requirements.adoc:11]:
```suggestion
NOTE: We have tested the plugin with Grafana version {grafana-version-tested}.
```

**Review by synqotik — APPROVED** [review]
Looks good! Thanks for updating the Requirements section.

We will address the build failure (CircleCI plugin signing issue) separately. It's also understood that there may be a few more small changes to some of these names and ids, but this takes care of most of them.

File: .circleci/config.yml [hunk 2]
```diff
@@ -346,7 +346,7 @@ jobs:
           command: |
             GRAFANA-PLUGIN_IMAGE_NAME="$(cat ./grafana-plugin-docker-image-name.txt)"
```
Comment (synqotik) [inline .circleci/config.yml:347]: fix to `GRAFANA_PLUGIN_IMAGE_NAME`

Also fix on lines 392-393

**Review by synqotik — CHANGES_REQUESTED** [review]
A couple small changes, otherwise looks good.

**Review by synqotik — APPROVED** [review]
looks good!

**Pull request merged**